aboutlibraries-compose/src/androidMain/kotlin/com/mikepenz/aboutlibraries/ui/compose/Libraries.kt

```diff
@@ -133,10 +135,10 @@ fun Libraries(
 }
 
 @Composable
-fun HtmlText(html: String, modifier: Modifier = Modifier) {
+fun HtmlText(html: String, color: Int, modifier: Modifier = Modifier) {
```

Comment:
@Nailik not 100% certain, but wouldn't it be better to pass in the `Color` here, and do the `toArgb` conversion only within the `setTextColor`? (as I suppose the `toArgb` is only required within the `View` world)

Reply:
would probably also work, but since the html view is not reused anywhere and toArgb is only invoked when the HtmlText is shown i don't see a problem here.
Actually how you like it, i can change it real quick.

Reply:
i changed it because then the usage of HtmlText feels more composable like

Reply:
I believe it would be cleaner. also while I know it's only used here, it may also be advised to provide a default value to the color for the web view, and have it ordered behind the modifier.

Reply:
done, i also added the parameter names to the call (both) because the one for the color is needed after reordering